src/commands/install.rs

```diff
@@ -157,12 +162,44 @@ fn archive(version: &Version) -> String {
     format!("ruby-{}.zip", version)
 }
 
-fn build_package(current_dir: &Path, installed_dir: &Path) -> Result<(), FrumError> {
-    debug!("./configure --with-openssl-dir={}", openssl_dir()?);
-    let configure = Command::new("sh")
+#[allow(clippy::unnecessary_wraps)]
+fn openssl_dir() -> Result<String, FrumError> {
+    #[cfg(target_os = "macos")]
+    return Ok(String::from_utf8_lossy(
+        &Command::new("brew")
+            .arg("--prefix")
+            .arg("openssl")
+            .output()
+            .map_err(FrumError::IoError)?
+            .stdout,
+    )
+    .trim()
+    .to_string());
+    #[cfg(not(target_os = "macos"))]
+    return Ok("/usr/local".to_string());
+}
+
+fn build_package(
+    current_dir: &Path,
+    installed_dir: &Path,
+    configure_opts: &Vec<String>,
```

Comment:
Could you use &[String]? `cargo clippy` causes a warning.

```suggestion
    configure_opts: &[String],
```